### PR TITLE
Feature/channels getters

### PR DESCRIPTION
**Pull request by DavidSubiros**
### What

- Added convenience functions:
  - LogErrors to log errors coming from the Errors channel
  - Channels getter
- Constructors only return errors if they are fatal errors. Initialisation errors are not returned at creation time (Initialise() can be called any time)

### How to review

- Make sure the changes make sense
- Unit tests should pass
- You may run the example (under cmd)

### Who can review

Anyone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The constructor tires to initialise the producer/consumer by creating the underl
 
 ### Initialisation
 
-A producer/consumer might have not been successfully initialised at creation time. If this is the case, you can always try to initialise it by calling `InitialiseSarama`. To validate the initialisation state, please call `IsInitialised`.
+A producer/consumer might have not been successfully initialised at creation time. If this is the case, you can always try to initialise it by calling `Initialise`. To validate the initialisation state, please call `IsInitialised`.
```
Comment (gedge) [inline README.md]:
```suggestion
A producer/consumer might not have been successfully initialised at creation time. If this is the case, you can always try to initialise it by calling `Initialise()`. To validate the initialisation state, please call `IsInitialised()`.
```

File: channels.go [hunk 2]
```diff
@@ -53,6 +59,21 @@ func (cCh *ConsumerGroupChannels) Validate() error {
 	return nil
 }
 
+// LogErrors creates a go-routine that waits on chErrors channel and logs any error received. It exits on chCloser channel event.
+// Provided context and errMsg will be used in the log Event.
+func (cCh *ConsumerGroupChannels) LogErrors(ctx context.Context, errMsg string) {
```
Comment (gedge) [inline channels.go]: not a huge fan of short var name `cCh`

File: channels.go [hunk 3]
```diff
@@ -53,6 +59,21 @@ func (cCh *ConsumerGroupChannels) Validate() error {
...
+	go func() {
+		for true {
```
Comment (gedge) [inline channels.go]:
```suggestion
		for {
```

File: channels.go [hunk 4]
```diff
@@ -77,6 +98,21 @@ func (pCh *ProducerChannels) Validate() error {
 	return nil
 }
 
+// LogErrors creates a go-routine that waits on chErrors channel and logs any error received. It exits on chCloser channel event.
+// Provided context and errMsg will be used in the log Event.
+func (pCh *ProducerChannels) LogErrors(ctx context.Context, errMsg string) {
+	go func() {
+		for true {
```
Comment (gedge) [inline channels.go]: same two issues as previous func

File: channels.go (hunk 2 above)
Comment (DavidSubiros) [reply channels.go]: I ca call them consumerChannels, producerChannels

Comment (gedge) [reply channels.go]: I think `consumerErrorChan` and `producerErrorChan` is better?

Comment (gedge) [reply channels.go]: sorry, should use plurals: `consumerErrorChans` and `producerErrorChans`

Comment (gedge) [reply channels.go]: ugh, ignore my last two comments, please - sorry

Comment (DavidSubiros) [reply channels.go]: no problem, thanks for the review!

**Pull request merged**